src/Domain/Migrations/Version20201005215932.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
```

Comment:
if i use `namespace App\Domain\Migrations;` when i try to run migration, doctrine doesn't found it...
Any idea @Rantignac ? I haven't search a lot for now.